hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/DistributionSetAssignmentResult.java

```diff
@@ -23,39 +19,35 @@
  */
 public class DistributionSetAssignmentResult extends AssignmentResult<Target> {
 
-    private final List<String> assignedTargets;
-    private final List<Action> actions;
-
+    private final List<? extends Action> assignedActions;
+    private final List<? extends Action> alreadyAssignedActions;
```

Comment:
are `alreadyAssignedActions` used anywhere in the code? or only in json payload? is there any tests?